examples/gno.land/r/matijamarjanovic/config/config.gno

```diff
@@ -0,0 +1,65 @@
+package config
+
+import (
+	"errors"
+	"std"
+)
+
+var (
+	main   std.Address // tony's main address
+	backup std.Address // backup address
+)
+
+func init() {
+	main = "g1n3s28tukh9fr37d44cucutsn22d4e8hw3666dv"
+}
+
+func Address() std.Address {
+	return main
+}
+
+func Backup() std.Address {
+	return backup
+}
+
+func SetAddress(a std.Address) error {
```

Comment:
very niptick: consider change parameter name `a` to more concrete name.

Reply:
Hi, please see fix here: ef98d67f51538ecdde5c496967a83b4d00e94673